contributing_to_docs/doc_guidelines.adoc

```diff
@@ -787,6 +784,27 @@ This renders as:
 +
 NOTE: Avoid using full command syntax inline with sentences.
 
+* When referring to a path to a location that the user has selected or created, treat the part of the path that the user chose as a replaceable value. For example:
++
+....
+Create a secret that contains the certificate and key in the `openshift-config` namespace:
+
+[source,terminal]
+----
+$ oc create secret tls <certificate> --cert=<path_to_certificate>/cert.crt --key=<path_to_key>/cert.key -n openshift-config
+----
+....
++
+This renders as:
++
+--
+> Create a secret that contains the certificate and key in the `openshift-config` namespace:
+>
+> ----
+> $ oc create secret tls <certificate> --cert=<path_to_certificate>/cert.crt --key=<path_to_key>/cert.key -n openshift-config
+> ----
+--
+
```

Comment:
**[05.05.2021] New**: specific instructions about paths as replaceable values

Reply:
Not 100% necessary, but in addition to a command example, could we also include a path example in a file? For example:

```
----
providerSpec:
  value:
    ...
    osDisk:
      diskSizeGB: 128
      managedDisk:
        diskEncryptionSet:
          id: <path_to_disk_encryption_set>/<disk_encryption_set_name>
...
----
```

vs what I currently have in a PR that I'm working on:

```
----
providerSpec:
  value:
    ...
    osDisk:
      diskSizeGB: 128
      managedDisk:
        diskEncryptionSet:
          id: /subscriptions/<subscription_id>/resourceGroups/<resource_group_name>/providers/Microsoft.Compute/diskEncryptionSets/<disk_encryption_set_name>
...
----
```